app/components/arcsi/Latest.vue

```diff
@@ -91,6 +91,7 @@ export default {
         return showslist
           .filter(item => item.play_date < this.getToday)
```

Comment:
This filtering is already implemented on Arcsi's side, we should check in the future if it is really necessary here

Reply:
Are you sure? Here the frontend uses `/all_without_items` which does not seem to do the filtering. It's only `/page` which does, no?

Reply:
O sorry, it's `/latest `which is used here and it's also filtered on arcsi side.

Reply:
I didn't know it uses, `/show/all_without_items`, in that case what I wrote is wrong. I just thought that it uses `/item/latest` because of the name of this file 😅

Reply:
Okay, then my assumption wasn't wrong :D

Reply:
https://github.com/lahmacunradio/frontend/issues/255 (frontend) and https://github.com/lahmacunradio/arcsi/issues/145 (arcsi).